app/javascript/src/components/Dashboard/Notes/Note.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "@bigbinary/neeto-icons";
+import { Typography, Dropdown, Tag, Tooltip, Avatar } from "neetoui";
+
+const Note = ({
+  id,
+  title,
+  description,
+  tag,
+  status,
+  created_at,
+  avatar,
+  setShowDeleteAlert,
+  setDeleteNoteId,
+  setIsOpen,
+}) => {
+  const deleteHelper = () => {
+    setIsOpen(true);
+    setShowDeleteAlert(true);
+    setDeleteNoteId(id);
+  };
+  return (
```

Comment:
Add a newline before the `return` statement since we are exiting the scope.

Reply:
This is resolved.